check_addon.py

```diff
@@ -320,11 +339,13 @@ def relative_path(file_path):
 
 def _logProblem(error_counter, problem_string):
     colorPrint("PROBLEM: %s" % problem_string, "31")
+    comments.append("PROBLEM: %s" % problem_string)
```

Comment:
Maybe we can have two lists, comments_problem and problems_warning. Then we could omit the PROBLEM: and WARNING: on each line and just write it one time followed by the respective comment list.